src/python_interface/CMakeLists.txt

```diff
@@ -164,11 +164,19 @@ set_target_properties(
     pyarts_cpp PROPERTIES LIBRARY_OUTPUT_DIRECTORY
     "${ARTS_BINARY_DIR}/python/src/pyarts/")
 
-nanobind_add_stub(
-    pyarts_cpp_stub
-    MODULE arts
-    OUTPUT ${ARTS_BINARY_DIR}/python/src/pyarts/arts.pyi
-    PYTHON_PATH ${ARTS_BINARY_DIR}/python/src/pyarts
-    DEPENDS pyarts_cpp
+include (ArtsNanobindStubs)
+arts_add_cpp_stubs(
+  constants
+  convert
+  disort
+  file
+  globals
+  hitran
+  interp
+  lbl
+  math
+  physics
+  predef
+  rtepack
+  zeeman
```

Comment:
This is going to be a nightmare.  Can you add a small pytest somewhere also that the list is exhaustive?  We can loop over all of pyarts.arts and below to check for if the named thing is a module, I think.  Well, I am doing it in the documentation file.

Reply:
I've added a test that outputs an error like this when a module is missing a `.pyi` file:
```
E       AssertionError: Missing stub file(s) for arts submodule(s):
E           hitran
E         Add them to the `arts_add_cpp_stubs` call in src/python_interface/CMakeLists.txt
```